sirepo/sim_data/flash.py

```diff
@@ -134,6 +129,11 @@ def proprietary_code_rpm(cls):
     def _compute_job_fields(cls, data, r, compute_model):
         return [r]
 
+    @classmethod
+    def _flash_file_hash(cls, data):
```

Comment:
@robnagler we talked about this some but never came up with an answer. Should the hash of the setup params be added to the filename? There is a tradeoff here between being able to delete the files (hard with the hash appended to the name) vs being able to identify the valid files (hard without the hash).

Reply:
FTR: Yes, it has to have the hash.